doc/12-icinga2-api.md

```diff
@@ -1585,25 +1585,29 @@ Example for all downtime events:
 
 #### <a id="icinga2-api-event-streams-type-checkresult"></a> Event Stream Type: CheckResult
 
-  Name 		| Type          | Description
-  --------------|---------------|--------------------------
-  type 	        | String        | Event type `CheckResult`.
-  timestamp     | Timestamp     | Unix timestamp when the event happened.
-  host	        | String        | [Host](09-object-types.md#objecttype-host) name.
-  service       | String        | [Service](09-object-types.md#objecttype-service) name. Optional if this is a host check result.
-  check\_result	| CheckResult   | Serialized [CheckResult](08-advanced-topics.md#advanced-value-types-checkresult) value type.
+  Name             | Type          | Description
+  -----------------|---------------|--------------------------
+  type             | String        | Event type `CheckResult`.
+  timestamp        | Timestamp     | Unix timestamp when the event happened.
+  host             | String        | [Host](09-object-types.md#objecttype-host) name.
+  service          | String        | [Service](09-object-types.md#objecttype-service) name. Optional if this is a host check result.
+  check\_result    | CheckResult   | Serialized [CheckResult](08-advanced-topics.md#advanced-value-types-checkresult) value type.
+  downtime\_depth  | Number        | Downtime information of checked object at the checked time.
+  acknowledgement  | Boolean       | Acknowledgment information of checked object at checked time.
```

Comment:
@N-o-X Do you agree that (1) "Downtime information" is a bit generic and (2) "of checked object at the checked time" is redundant? (same for ack)

Reply:
@Al2Klimov Is (1) "Downtime depth at the checked time." and (2) "Is the object has already been acknowledged when it is in a problem status." ok for you?

Reply:
1. Downtime depth
2. Whether the object is acknowledged

Reply:
@Al2Klimov I changed it.